experimental/FTheoryTools/src/AbstractFTheoryModels/methods.jl

```diff
@@ -624,6 +630,40 @@ function set_zero_section(m::AbstractFTheoryModel, desired_value::Vector{String}
   set_attribute!(m, :zero_section => [f(eval_poly(l, R)) for l in desired_value])
 end
 
+function set_gauge_algebra(m::AbstractFTheoryModel, algebras::Vector{String})
+gauge_algebras = [];
+C = algebraic_closure(QQ);
+for i in 1:length(algebras)
+    g = algebras[i];
```

Comment:
Do you use `i` in here? If not just do
```suggestion
for g in algebras
```